Classes/Public/Manager/OCTManagerConstants.h

```diff
@@ -44,17 +45,29 @@ typedef NS_ENUM(NSInteger, OCTMessageFileType) {
 
 typedef NS_ENUM(NSInteger, OCTMessageCallEvent) {
     /**
-     * Call dialing
+     * Call was answered.
      */
-    OCTMessageCallEventDial,
+    OCTMessageCallEventAnswered,
 
     /**
-     * Call was missed.
+     * Call was unanswered.
      */
-    OCTMessageCallEventMissed,
+    OCTMessageCallEventUnanswered,
+};
+
+typedef NS_ENUM(NSInteger, OCTCallStatus) {
+    /**
+     * Call is currently ringing.
+     */
+    OCTCallStatusRinging,
+
+    /**
+     * Call is currently dialing a chat.
+     */
+    OCTCallStatusDialing,
 
     /**
-     * Call has ended.
+     * Call is currently active in session.
      */
-    OCTMessageCallEventEnd,
+    OCTCallStatusActive,
```

Comment:
Well `paused` state just means muting incoming/outgoing audio or video. I wouldn't call it "paused" since the user will be technically still be in a call. Which I think is why I didn't originally didn't stop or start the timer. What do you think is better?